commit bfbd7b598bab389183c6d8d4852555c41e99b23e (refs/changes/55/3655/1)
Author: Toomas Soome <tsoome@me.com>
Date:   2018-03-16T11:49:34-05:00 (1 year, 7 months ago)
    
    9173 smbsrv: we do not build all
    Reviewed by: Andy Fiddaman <omnios@citrus-it.co.uk>
    Reviewed by: Yuri Pankov <yuripv@yuripv.net>
    Approved by: Gordon Ross <gwr@nexenta.com>

diff --git a/usr/src/lib/smbsrv/Makefile b/usr/src/lib/smbsrv/Makefile
index 9ba411e8ce..4213133a56 100644
--- a/usr/src/lib/smbsrv/Makefile
+++ b/usr/src/lib/smbsrv/Makefile
@@ -34,9 +34,9 @@ SUBDIRS = \
 	libsmbns \
 	libsmbrp
 
+include ./Makefile.subdirs
+
 libmlrpc: libsmb
 libsmbns: libsmb
 libmlsvc: libsmb libmlrpc libsmbns
 libfksmbsrv: libsmb
-
-include ./Makefile.subdirs
